  INFO root [0x0000943c] [2018-02-03 18:30:37,111] - 
  INFO root [0x0000943c] [2018-02-03 18:30:37,112] - -----------------------------------------------------------------------------------------


  INFO root [0x0000943c] [2018-02-03 18:30:37,113] - Resmgr::initialize: KBE_ROOT=E:/Ambicour/DeepSpace/ServerAssets/
  INFO root [0x0000943c] [2018-02-03 18:30:37,113] - Resmgr::initialize: KBE_RES_PATH=E:/Ambicour/DeepSpace/ServerAssets/kbe/res/;E:/Ambicour/DeepSpace/ServerAssets/assets/;E:/Ambicour/DeepSpace/ServerAssets/assets/scripts/;E:/Ambicour/DeepSpace/ServerAssets/assets/res/
  INFO root [0x0000943c] [2018-02-03 18:30:37,113] - Resmgr::initialize: KBE_BIN_PATH=E:/Ambicour/DeepSpace/ServerAssets/kbe/bin/server/
  INFO root [0x0000943c] [2018-02-03 18:30:37,196] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.8.230 ( 192.168.8.230:63037 )
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - NetworkInterface::initialize(INTERNAL): address 192.168.8.230:63037, SOMAXCONN=5.
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - server-configs:
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - 	gameUpdateHertz : 10
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - 	defaultViewRadius : 80
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - 	defaultViewHysteresisArea : 5
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - 	entryScriptFile : kbemain
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - 	internalAddr : 192.168.8.230:63037
  INFO root [0x0000943c] [2018-02-03 18:30:37,197] - 	componentID : 8001
  INFO root [0x0000943c] [2018-02-03 18:30:38,703] - ---- cellapp Version: 1.1.0. ScriptVersion: 0.1.0. Protocol: 2C9309937A59A1F67DE84AFADAE427D6. Config: Debug X86. Built: 18:10:30 Feb  2 2018. AppID: 8001. UID: 10103. PID: 24676 ----
  INFO root [0x0000943c] [2018-02-03 18:30:38,703] - Server : Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x0000943c] [2018-02-03 18:30:38,703] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000943c] [2018-02-03 18:30:38,704] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,843] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,843] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,844] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,847] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,848] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,848] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,848] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000943c] [2018-02-03 18:30:38,896] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000943c] [2018-02-03 18:30:38,943] - Script::install(): is successfully, Python=(3.4.2 (default, Feb  2 2018, 18:09:04) [MSC v.1911 32 bit (Intel)])!
  INFO root [0x0000943c] [2018-02-03 18:30:38,943] - Script::install(): KBExtra is successfully!
  INFO root [0x0000943c] [2018-02-03 18:30:38,964] - TelnetServer server is running on port 50000
  INFO root [0x0000943c] [2018-02-03 18:30:39,350] - ---- cellapp is running ----
 DEBUG root [0x0000943c] [2018-02-03 18:30:39,350] - Components::process(): Request for the process of identity...
  WARN root [0x0000943c] [2018-02-03 18:30:45,668] - EntityApp::checkTickPeriod: tick took 6317% (6.32 seconds)!
  INFO root [0x0000943c] [2018-02-03 18:30:46,715] - Components::findComponents: find logger(1)...
  INFO root [0x0000943c] [2018-02-03 18:30:46,813] - Components::findComponents: found logger, addr:192.168.8.230:63042
  INFO root [0x0000943c] [2018-02-03 18:30:46,813] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x0000943c] [2018-02-03 18:30:46,813] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000943c] [2018-02-03 18:30:46,814] - Components::findComponents: find dbmgr(2)...
  INFO root [0x0000943c] [2018-02-03 18:30:46,919] - Components::findComponents: found dbmgr, addr:192.168.8.230:63039
  INFO root [0x0000943c] [2018-02-03 18:30:46,919] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x0000943c] [2018-02-03 18:30:46,919] - Components::addComponent[dbmgr], uid=10103, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000943c] [2018-02-03 18:30:46,919] - Components::findComponents: found dbmgr, addr:192.168.8.230:63039
  WARN root [0x0000943c] [2018-02-03 18:30:47,920] - Forwarding logs to logger[192.168.8.230:63042/0/0/0]...
 ERROR root [0x0000943c] [2018-02-03 18:30:56,836] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x0000943c] [2018-02-03 18:30:56,846] - The following logs sent to logger failed:
  WARN root [0x0000943c] [2018-02-03 18:30:56,904] - ==> [2018-02-03 18:30:56 836] TCPPacketReceiver::processPendingEvents(192.168.8.230:63042): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000943c] [2018-02-03 18:30:56,904] - ==> [2018-02-03 18:30:56 837] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x0000943c] [2018-02-03 18:30:56,976] - TCPPacketReceiver::processPendingEvents(192.168.8.230:63039): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000943c] [2018-02-03 18:30:56,980] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x0000943c] [2018-02-03 18:30:57,058] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x0000943c] [2018-02-03 18:30:57,059] - Shutdowner::onShutdown: shutting down
  INFO root [0x0000943c] [2018-02-03 18:30:57,133] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,182] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,182] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,287] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,287] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,287] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,287] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,288] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,288] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,288] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,288] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x0000943c] [2018-02-03 18:30:57,318] - Script::uninstall(): is successfully!
 DEBUG root [0x0000943c] [2018-02-03 18:30:57,323] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001b58] [2018-02-03 18:31:02,315] - 
  INFO root [0x00001b58] [2018-02-03 18:31:02,352] - -----------------------------------------------------------------------------------------


  INFO root [0x00001b58] [2018-02-03 18:31:02,352] - Resmgr::initialize: KBE_ROOT=E:/Ambicour/DeepSpace/ServerAssets/
  INFO root [0x00001b58] [2018-02-03 18:31:02,353] - Resmgr::initialize: KBE_RES_PATH=E:/Ambicour/DeepSpace/ServerAssets/kbe/res/;E:/Ambicour/DeepSpace/ServerAssets/assets/;E:/Ambicour/DeepSpace/ServerAssets/assets/scripts/;E:/Ambicour/DeepSpace/ServerAssets/assets/res/
  INFO root [0x00001b58] [2018-02-03 18:31:02,353] - Resmgr::initialize: KBE_BIN_PATH=E:/Ambicour/DeepSpace/ServerAssets/kbe/bin/server/
  INFO root [0x00001b58] [2018-02-03 18:31:02,359] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.8.230 ( 192.168.8.230:63084 )
  INFO root [0x00001b58] [2018-02-03 18:31:02,359] - NetworkInterface::initialize(INTERNAL): address 192.168.8.230:63084, SOMAXCONN=5.
  INFO root [0x00001b58] [2018-02-03 18:31:02,359] - server-configs:
  INFO root [0x00001b58] [2018-02-03 18:31:02,359] - 	gameUpdateHertz : 10
  INFO root [0x00001b58] [2018-02-03 18:31:02,359] - 	defaultViewRadius : 80
  INFO root [0x00001b58] [2018-02-03 18:31:02,359] - 	defaultViewHysteresisArea : 5
  INFO root [0x00001b58] [2018-02-03 18:31:02,359] - 	entryScriptFile : kbemain
  INFO root [0x00001b58] [2018-02-03 18:31:02,360] - 	internalAddr : 192.168.8.230:63084
  INFO root [0x00001b58] [2018-02-03 18:31:02,360] - 	componentID : 8001
  INFO root [0x00001b58] [2018-02-03 18:31:03,894] - ---- cellapp Version: 1.1.0. ScriptVersion: 0.1.0. Protocol: 2C9309937A59A1F67DE84AFADAE427D6. Config: Debug X86. Built: 18:10:30 Feb  2 2018. AppID: 8001. UID: 10103. PID: 23116 ----
  INFO root [0x00001b58] [2018-02-03 18:31:03,895] - Server : Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00001b58] [2018-02-03 18:31:03,895] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001b58] [2018-02-03 18:31:03,895] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,034] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,035] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,035] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,039] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,040] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,040] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,040] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,091] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001b58] [2018-02-03 18:31:04,147] - Script::install(): is successfully, Python=(3.4.2 (default, Feb  2 2018, 18:09:04) [MSC v.1911 32 bit (Intel)])!
  INFO root [0x00001b58] [2018-02-03 18:31:04,147] - Script::install(): KBExtra is successfully!
  INFO root [0x00001b58] [2018-02-03 18:31:04,172] - TelnetServer server is running on port 50000
  INFO root [0x00001b58] [2018-02-03 18:31:04,643] - ---- cellapp is running ----
 DEBUG root [0x00001b58] [2018-02-03 18:31:04,643] - Components::process(): Request for the process of identity...
  WARN root [0x00001b58] [2018-02-03 18:31:10,992] - EntityApp::checkTickPeriod: tick took 6350% (6.35 seconds)!
  INFO root [0x00001b58] [2018-02-03 18:31:12,043] - Components::findComponents: find logger(1)...
  INFO root [0x00001b58] [2018-02-03 18:31:12,106] - Components::findComponents: found logger, addr:192.168.8.230:63079
  INFO root [0x00001b58] [2018-02-03 18:31:12,106] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00001b58] [2018-02-03 18:31:12,106] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001b58] [2018-02-03 18:31:12,107] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001b58] [2018-02-03 18:31:12,212] - Components::findComponents: found dbmgr, addr:192.168.8.230:63085
  INFO root [0x00001b58] [2018-02-03 18:31:12,212] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00001b58] [2018-02-03 18:31:12,212] - Components::addComponent[dbmgr], uid=10103, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001b58] [2018-02-03 18:31:12,213] - Components::findComponents: found dbmgr, addr:192.168.8.230:63085
  WARN root [0x00001b58] [2018-02-03 18:31:13,213] - EntityApp::checkTickPeriod: tick took 1270% (1.27 seconds)!
  WARN root [0x00001b58] [2018-02-03 18:31:13,214] - Forwarding logs to logger[192.168.8.230:63079/0/0/0]...
 ERROR root [0x00001b58] [2018-02-03 18:33:27,510] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001b58] [2018-02-03 18:33:27,522] - The following logs sent to logger failed:
  WARN root [0x00001b58] [2018-02-03 18:33:27,654] - ==> [2018-02-03 18:33:27 508] TCPPacketReceiver::processPendingEvents(192.168.8.230:63079): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001b58] [2018-02-03 18:33:27,654] - ==> [2018-02-03 18:33:27 511] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001b58] [2018-02-03 18:33:27,655] - EntityApp::checkTickPeriod: tick took 211% (0.21 seconds)!
  WARN root [0x00001b58] [2018-02-03 18:33:27,655] - TCPPacketReceiver::processPendingEvents(192.168.8.230:63085): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001b58] [2018-02-03 18:33:27,656] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001b58] [2018-02-03 18:33:27,686] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001b58] [2018-02-03 18:33:27,689] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001b58] [2018-02-03 18:33:27,749] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,798] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,798] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,937] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,937] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,937] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,937] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,937] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,937] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,938] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001b58] [2018-02-03 18:33:27,938] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001b58] [2018-02-03 18:33:28,030] - Script::uninstall(): is successfully!
 DEBUG root [0x00001b58] [2018-02-03 18:33:28,040] - ThreadPool::destroy(): starting size 2.
